### ISSUE TITLE
REPL requires

### ISSUE THREAD

**Issue by mfikes** [mfikes/replete#20]
Add the normal slew of namespaces that ClojureScript REPLs get by default, and alias things in like `pprint`.

**Comment by skrat** [comment]
I think `cljs.pprint/pprint` should be the default for printing return values, amirite?

**Comment by mfikes** [comment]
@skrat Yes, `pprint` and `pp` should be available in Replete as this is the default for normal ClojureScript REPLs: https://github.com/clojure/clojurescript/blob/2924c4880c05208beb0f321e8e4e63b4cb1c45f3/src/main/clojure/cljs/repl.cljc#L739
